tutorials/vo/conesearch_tutorial.ipynb

```diff
@@ -127,7 +127,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
```

Comment:
Note: I have no idea why this part changed. I edited the notebook using Jupyter.